config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
```

Comment:
just checked and your last example does pass.  Yay!

I've gotten frustrated in the past when I've had to muck about multiple times with alignment for parens, arrays, hashes, arguments, etc.  to no cognitive gain but perhaps with the other relevant alignment rules, we have gotten to a solution I can live with.